### DISCUSSION TITLE
Prevent dupes from RSS feeds

### DISCUSSION THREAD

**Discussion by ghost** [Flexget/Flexget#3926] [Q&A]
Any way to prevent this from happening?

```
2024-01-23 06:58:41 VERBOSE  task          TOL             ACCEPTED: `Movie 2023 2160p AMZN WEB-DL DDP5 1 Atmos HDR H 265-FLUX` by regexp plugin because regexp '(web|remux|hevc|1080|4k|2160)' matched field 'title'

2024-01-23 13:05:44 VERBOSE  task          FLT             ACCEPTED: `Movie.2023.2160p.AMZN.WEB-DL.DDP5.1.Atmos.HDR.H.265-FLUX` by regexp plugin because regexp '(.*)' matched field 'title'
```

I replace the title with `Movie`. I guess it doesn't see it as a dupe because one uses spaces and the other dots. They are the same movie though, the same file. Idential titles if you replace dots with spaces or vice versa.

**Comment by gazpachoking** [comment]
Try [seen_movies](https://flexget.com/en/Plugins/seen_movies) plugin

**Comment by BrutuZ** [comment]
Slightly unrelated: Are these RegEx real or trimmed for sharing?
`accept_all` plug-in could replace `.*` and the `quality` plugin handle the filtering?

**Reply by ghost** [reply]
I should do accept_all on these indeed. But the dupe is my main issue.

**Reply by gazpachoking** [reply]
I updated my comment to spell the plugin right and link it. I think that should take care of your issue. If it doesn't work on its own, adding a lookup plugin like imdb_lookup, or trakt_lookup will cause it to determine dupes by their imdb or trakt id that is looked up rather than relying on raw string parsing.

**Reply by ghost** [reply]
What about the manipulate plugin? Replace space with a dot or whatever.

**Reply by gazpachoking** [reply]
Did you try seen_movies? It already tries to normalize the title if any looked up ids aren't available.
